src/documentation/print-linting-and-testing-wiki.ts

```diff
@@ -0,0 +1,186 @@
+import { codeBlock } from './doc-util/doc-code';
+import { FlowrDockerRef, FlowrGithubBaseRef, FlowrSiteBaseRef, getFilePathMd, RemoteFlowrFilePathBaseRef } from './doc-util/doc-files';
+
+function getText() {
+	return `
+For the latest code-coverage information, see [codecov.io](https://app.codecov.io/gh/flowr-analysis/flowr), 
+for the latest benchmark results, see the [benchmark results](${FlowrSiteBaseRef}/wiki/stats/benchmark) wiki page.
+
+- [Testing Suites](#testing-suites)
+  - [Functionality Tests](#functionality-tests)
+    - [Test Structure](#test-structure)
+    - [Writing a Test](#writing-a-test)
+    - [Running Only Some Tests](#running-only-some-tests)
+  - [Performance Tests](#performance-tests)
+  - [Oh no, the tests are slow](#oh-no-the-tests-are-slow)
+  - [Testing Within Your IDE](#testing-within-your-ide)
+    - [Using Visual Studio Code](#vs-code)
+    - [Using WebStorm](#webstorm)
+- [CI Pipeline](#ci-pipeline)
+- [Linting](#linting)
+  - [Oh no, the linter fails](#oh-no-the-linter-fails)
+  - [License Checker](#license-checker)
+
+## Testing Suites
+
+Currently, flowR contains two testing suites: one for [functionality](#functionality-tests) and one for [performance](#performance-tests). We explain each of them in the following.
+A more general test experience is granted by \`npm run checkup\` which will include the construction of the docker image, the generation of the wiki pages, and the linter.
+
+### Functionality Tests
+
+The functionality tests represent conventional unit (and depending on your terminology component/api) tests.
+We use [vitest](https://vitest.dev/) as our testing framework.
+You can run the tests by issuing:
+
+${codeBlock('shell', 'npm run test')}
+
+Within the commandline,
+this should automatically drop you into a watch mode which will automatically re-run the tests if you change the code.
+If, at any time there are too many errors, you can use \`--bail=<value>\` to stop the tests after a certain number of errors.
+For example:
+
+${codeBlock('shell', 'npm run test -- --bail=1')}
+
+If you want to run the tests without the watch mode, you can use:
+
+${codeBlock('shell', 'npm run test -- --no-watch')}
+
+To run all tests, including a coverage report and label summary, run: 
+
+${codeBlock('shell', 'npm run test-full')}
+
+However, depending on your local R version, your network connection and potentially other factors, some tests may be skipped automatically as they don’t apply to your current system setup 
+(or can't be tested with the current prerequisites). 
+Each test can specify such requirements as part of the \`TestConfiguration\`, which is then used in the \`test.skipIf\` function of _vitest_.
+It is up to the [ci](#ci-pipeline) to run the tests on different systems to ensure that those tests are ensured to run.
+
+#### Test Structure
+
+All functionality tests are to be located under [test/functionality](${RemoteFlowrFilePathBaseRef}test/functionality).
```

Comment:
Did i add the trailing `/` to the variable?

Reply:
`RemoteFlowrFilePathBaseRef` and `FlowrWikiBaseRef` have the trailing slashes :)) the rest don't

✨ consistency